azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/VirtualMachineScaleSetNicIpConfiguration.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.network;
+
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.network.implementation.NetworkInterfaceIPConfigurationInner;
+import com.microsoft.azure.management.network.model.HasPrivateIpAddress;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ChildResource;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.HasSubnet;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+
+import java.util.List;
+
+/**
+ * An IP configuration in a network interface associated with a virtual machine
+ * scale set.
+ */
+@Fluent
+public interface VirtualMachineScaleSetNicIpConfiguration extends
```

Comment:
similarly here, can we have a shared base interface for regular NIC IP configs and this?